Only unserialize if it is not the SlackRTM or Console driver

christophrumpel opened the pull request:
Fixing #386 where BotMan tries to unserialise a not serialized callback. You wanted to reuse the `unserializeClosure` method from the HandlesConversations trait, but I wasn't sure how to use that in the Conversation class. Feel free to tell how to optimise it.

mpociot commented:
As discussed on Slack, please change the check from the driver name to a new method on the driver class/interface.

christophrumpel commented:
Ok I changed the code to use a method from the abstract driver class. Only problem is the ConsoleDriver from the botman tinker package is not extending from this class. So we need to add this method to the botman-tinker package too. Guess there was a reason why the console class and SlackRTM class are not extending the Driver class?

mpociot commented:
Yes. The abstract class is only for HTTP based drivers - thus will be renamed to HttpDrover in 2.0. 
so we need this new method on the driver interface too :)

christophrumpel commented:
Ah I see hehe. Yes of course. Added it to the interface and included the method in the needed drivers. Also added a basic test for these drivers to check if it false. For all the the "not-http-drivers" I set the the value to `false`. Guess we still need to add the method to the ConsoleDriver in the botman-tinker repo.

mpociot commented:
I'll close this, as this is a duplicate of #400 
Once #400 is merged, I'll take care of backporting it to the 1.5 branch.
